rebar.lock

```diff
@@ -9,7 +9,7 @@
  {<<"covertool">>,
   {git,"https://github.com/idubrov/covertool",
        {ref,"be45cabc5dc28480176172425793923b6cde428d"}},
-  2},
+  1},
```

Comment:
I believe it is ordering used internally by rebar - this is a generated file though, so shouldn't be a problem.